templates/configmap.yaml

```
@@ -0,0 +1,12 @@
+{{- if (and (not .Values.useExternalConfig) (.Values.useConfigMap)) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "cortex.fullname" . }}
```

```suggestion
  name: {{ template "cortex.fullname" . }}-config
```
tiny improvement idea